src/components/v5/common/Fields/Datepicker/partials/DatepickerCustomRangeInput/DatepickerCustomRangeInput.tsx

```diff
@@ -0,0 +1,37 @@
+import format from 'date-fns/format';
+import React from 'react';
+
+import { formatText } from '~utils/intl.ts';
+import InputBase from '~v5/common/Fields/InputBase/index.ts';
+
+import { type DatepickerCustomRangeInputProps } from './types.ts';
+
+const DatepickerCustomRangeInput = React.forwardRef<
+  HTMLInputElement,
+  DatepickerCustomRangeInputProps
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+>(({ onClick, onFocus, startDate, endDate, dateFormat }, ref) => {
+  return (
+    <div className="w-full flex items-center justify-between gap-2">
+      <InputBase
+        placeholder={formatText({ id: 'calendar.from' })}
+        readOnly
+        onClick={onClick}
+        onFocus={onFocus}
+        value={startDate ? format(startDate, dateFormat) : ''}
+      />
+      <span className="flex justify-center items-center flex-shrink-0 text-gray-900 text-2 w-6 text-center">
+        -
+      </span>
+      <InputBase
+        placeholder={formatText({ id: 'calendar.to' })}
+        readOnly
+        onClick={onClick}
+        onFocus={onFocus}
+        value={endDate ? format(endDate, dateFormat) : ''}
+      />
```

Comment:
Can't really find a spec for this, so maybe @melyndav can chime in, but should the inputs allow users to change the values manually, as to me they looks more for display purposes only 

![Screenshot from 2024-02-16 18-56-56](https://github.com/JoinColony/colonyCDapp/assets/1193222/c7a12d17-4461-42f8-97d3-2409ee559539)
![Screenshot from 2024-02-16 18-57-03](https://github.com/JoinColony/colonyCDapp/assets/1193222/e5d77872-896b-41af-89c5-ae2dd59e8448)

This is especially relevant since the date format is quite specific, making it nearly impossible for the layman user to actually write this by hand

So what I'm saying is that we still keep the inputs, we just disable them